.travis.yml

```diff
@@ -15,7 +15,7 @@ python:
 env:
   global:
     - CI_CONCURRENT_JOBS=1
-    - OPENSHIFT_ANSIBLE_COMMIT=master
+    - OPENSHIFT_ANSIBLE_COMMIT=1febf0b20de622d358dc890417dc4cda7c3f71a9
```

Comment:
The CI's running now so I'll let it run its course. We'd have to re-run it with the tag afterwards (to make sure that fixes it) so if you don't mind, I'd just leave this one be and I'll make sure to use tags here in the future.